api/knexfile.js

```diff
@@ -0,0 +1,57 @@
+// Update with your config settings.
+
+/**
+ * @type { Object.<string, import("knex").Knex.Config> }
+ */
+
+const env = require('../config');
```

Comment:
knexfile에서 configure 하지 않아도 될 것 같습니다.
아래 local, development, production config 코드는 없어도 됩니다.

```suggestion
const config = require('../config');
module.exports = config.mysql;
```